Source/OxyPlot.ImageSharp/PngRenderingContext.cs

```diff
@@ -75,42 +87,286 @@ public void Save(Stream output)
+        {
+            switch (edgeRenderingMode)
+            {
+                case EdgeRenderingMode.Automatic when RenderContextBase.IsStraightLine(screenPoints):
+                case EdgeRenderingMode.Adaptive when RenderContextBase.IsStraightLine(screenPoints):
+                case EdgeRenderingMode.PreferSharpness:
+                    var snapOffset = this.GetSnapOffset(strokeThickness, edgeRenderingMode);
+                    return screenPoints.Select(p => this.ConvertSnap(p, snapOffset));
+                default:
+                    return screenPoints.Select(this.Convert);
+            }
+        }
+
+        /// <summary>
+        /// Gets the <see cref="RectangleF"/> that should actually be rendered from the <see cref="OxyRect"/>, taking into account DPI scaling and snapping if necessary.
+        /// </summary>
+        /// <param name="rect">The rectangle.</param>
+        /// <param name="strokeThickness">The stroke thickness.</param>
+        /// <param name="edgeRenderingMode">The edge rendering mode.</param>
+        /// <returns>The actual rectangle.</returns>
+        private RectangleF GetActualRect(OxyRect rect, double strokeThickness, EdgeRenderingMode edgeRenderingMode)
+        {
+            switch (edgeRenderingMode)
+            {
+                case EdgeRenderingMode.PreferGeometricAccuracy:
+                case EdgeRenderingMode.PreferSpeed:
+                    return this.Convert(rect);
+                default:
+                    var actualThickness = this.GetActualThickness(strokeThickness, edgeRenderingMode);
+                    var snapOffset = GetSnapOffset(actualThickness);
+                    return this.ConvertSnap(rect, snapOffset);
+            }
+        }
+
+        /// <summary>
+        /// Gets the <see cref="RectangleF"/>s that should actually be rendered from the list of <see cref="OxyRect"/>s, taking into account DPI scaling and snapping if necessary.
+        /// </summary>
+        /// <param name="rects">The rectangles.</param>
+        /// <param name="strokeThickness">The stroke thickness.</param>
+        /// <param name="edgeRenderingMode">The edge rendering mode.</param>
+        /// <returns>The actual rectangles.</returns>
+        private IEnumerable<RectangleF> GetActualRects(IEnumerable<OxyRect> rects, double strokeThickness, EdgeRenderingMode edgeRenderingMode)
+        {
+            switch (edgeRenderingMode)
+            {
+                case EdgeRenderingMode.PreferGeometricAccuracy:
+                case EdgeRenderingMode.PreferSpeed:
+                    return rects.Select(this.Convert);
+                default:
+                    var actualThickness = this.GetActualThickness(strokeThickness, edgeRenderingMode);
+                    var snapOffset = GetSnapOffset(actualThickness);
+                    return rects.Select(rect => this.ConvertSnap(rect, snapOffset));
+            }
+        }
```

Comment:
These appear to be unused, so can be removed.

Reply:
Should probably override `DrawRectangles` at some point, but for now I will just remove these.